addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -733,6 +733,20 @@ export class OdooEditor extends EventTarget {
         step.mutations = step.mutations.slice(0, until);
         this._toRollback = false;
     }
+    /**
+     * Undo the current non-recorded draft step.
+     */
+    historyRevertDraft() {
```

Comment:
I would call it `historyRevertCurrentStep`.

Reply:
@Goaman Suggested change done